app/scripts/views/sign_up.js

```diff
@@ -61,8 +63,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
 
       this._bouncedEmail = this.ephemeralMessages.get('bouncedEmail');
-
-      return FormView.prototype.beforeRender.call(this);
+      getNow(['passwordcheck'], function (PasswordCheck) {
+        self.passwordcheck = new PasswordCheck();
```

Comment:
quick: `passwordcheck` and `passwordCheck`. What's the difference? It took me 3 readings of the code to realize these two names are so close to each other.

rename `passwordCheck` to `checkPassword`. I like functions to start with a verb. Rename `passwordcheck` to `_checkPassword`. It could probably use even a different name than that. At least with the underscore there is some immediately recognizable difference.